spec/models/comment_spec.rb

```diff
@@ -0,0 +1,6 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  let(:tom) { User.create(name: 'Tom', posts_counter: 0) }
+  let(:post) { Post.create(title: 'Title', comments_counter: 0, likes_counter: 0, author: tom) }
+end
```

Comment:
- [ ] What are you testing here? Please 🙏🏽 validate the comment when it should have a text, it is valid with an author and a post. It is a requirement for this milestone 👇🏽 

> Add unit specs for all of your models' methods and validations.

Reply:
I was so absorbed in that moment that I guess I skipped it. I will do that. Thank you.